crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -236,3 +236,28 @@ def foo():
         "b",
     ]:
         pass
+
+
+match foo:
+    case 1:
+        y = 0
+    case (1):
+        y = 1
```

Comment:
Could you add a double parentheses test case?
```suggestion
    case (1):
        y = 1
    case (("a")):
        y = 1
```